content/get-started/06_bind_mounts.md

```diff
@@ -45,10 +45,17 @@ work.
 >
 > If you use Windows and want to use Git Bash to run Docker commands, see [Working with Git Bash](../desktop/troubleshoot/topics.md#working-with-git-bash) for syntax differences.
 
-1. Open a terminal and change directory to the `getting-started-app`
+1. In the Docker Desktop Dashboard, view the file sharing setting. This setting
```

Comment:
should we add something like "make sure that your `getting-started-app` project is located under a file share" (really not sure how to phrase it but you know what I mean 😆)